src/relay/backend/build_module.cc

```diff
@@ -438,13 +442,19 @@ class RelayBuildModule : public runtime::ModuleNode {
 
     auto lowered_funcs = graph_codegen_->GetLoweredFunc();
     if (lowered_funcs.size() == 0) {
-      LOG(WARNING) << "no lowered funcs exist in the compiled module";
-    } else {
-      ret_.mod = tvm::build(
-        lowered_funcs,
-        target_host_,
-        BuildConfig::Current());
+      LOG(WARNING) << "No lowered funcs exist in the compiled module, "
```

Comment:
Do we need to retain this warning? With external codegen, having no lowered funcs can be a perfectly normal mode of operation.

Reply:
Thanks @mbarrett97 , I've removed that log.